boards/veer_el2_sim/src/io.rs

```diff
@@ -1,48 +1,28 @@
 // Licensed under the Apache License, Version 2.0 or the MIT License.
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 // Copyright Tock Contributors 2022.
+// Copyright (c) 2024 Antmicro <www.antmicro.com>
```

Comment:
I don't think you should add this to an existing file

Reply:
Our general policy is that people are free to _add_ copyright information for non-trivial changes:

> Substantial updates to an artifact MAY add additional copyright notices to an
artifact. In general, modifications to a file are expected to retain existing
copyright notices.

Reply:
Yeah, but to me it's a trivial change. It's just removing some code and renaming the file

Reply:
Alright, I've removed it.